internal/fs/fs.go

```diff
@@ -1309,10 +1309,24 @@ func (fs *fileSystem) StatFS(
 	return
 }
 
+// isolateContextFromParentContext creates a copy of the parent context which is
+// not cancelled when parent context is cancelled.
+func isolateContextFromParentContext(ctx context.Context) (context.Context, context.CancelFunc) {
```

Comment:
Given that this method is now abstracted from business logic of fs.go, should we add this and test cases in https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/util/util.go?